src/middlewares.js

```diff
@@ -0,0 +1,36 @@
+const { verify } = require('jsonwebtoken');
+
+exports.verify = (req, res, next) => {
+  if (!req.headers.cookie || !req.headers.cookie.includes('jwt')) {
+    next();
+  } else {
+    const jwt = req.cookies.jwt;
+    if (jwt) {
+      verify(jwt, process.env.SECRET, (err, jwt) => {
+        if (err) {
+          res.send('Fail');
+        } else {
+          req.is_user = 'user';
+          if (jwt.type === 'person') {
```

Comment:
why are you assigning values for req.is_user? Handling authentication should be done 100% through the cookies and the jwt.